ProvisionPullClient.ps1

```diff
@@ -17,72 +28,28 @@
     ProvisionPullClient.ps1 e7d38156-02b2-42d3-ad0a-4457fe8cf380 https://pullserver.example.com:8080/PSDSCPullServer.svc
 #>
 
-Param (
-    # Parameter that defines the configuration ID to be used by the client
-    [Parameter(Mandatory=$True)]
-    [string]$ConfigurationIDGUID,
-
-    # Parameter that defines the Pull server the client will use
-    [Parameter(Mandatory=$True)]
-    [string]$PullServerURL
-)
 
 # Temp folder used for outputting the MOF files so they are in a known location
-If (!(Test-Path 'C:\Temp'))
+If (!(Test-Path "$PSScriptRoot\DSC"))
 {
-    New-Item 'C:\Temp' -ItemType Directory
+    New-Item "$PSScriptRoot\DSC" -ItemType Directory
 }
 
-# Work from the temp folder
-Set-Location 'C:\Temp'
-
 # Use switch to configure the pull client dependant on version of PS installed
 # PS v5 https://msdn.microsoft.com/en-us/powershell/dsc/pullclientconfigid
 # PS v4 https://msdn.microsoft.com/en-us/powershell/dsc/pullclientconfigid4
-Switch ($PSVersionTable.PSVersion.Major){
-    5 {
-        [DSCLocalConfigurationManager()]
-        configuration PullClientConfigID
-        {
-            Node localhost
-            {
-                Settings
-                {
-                    RefreshMode = 'Pull'
-                    ConfigurationID = $ConfigurationIDGUID
-                    RefreshFrequencyMins = 30 
-                    RebootNodeIfNeeded = $true
-                    ConfigurationMode = "ApplyAndAutoCorrect"
-                }
-                ConfigurationRepositoryWeb PullSrv
-                {
-                    ServerURL = $PullServerURL
-                }      
-            }
-        }
-        PullClientConfigID
 
-        Set-DscLocalConfigurationManager -Path ".\PullClientConfigID"
+
+Switch ($PSVersionTable.PSVersion.Major)
+{
+    5 {
+         Invoke-WebRequest https://raw.githubusercontent.com/justeat/PowerShellDSCUtils/master/Version5DSC.ps1 -OutFile "$PSScriptRoot\DSC\Version5DSC.ps1"
```

Comment:
Hardcoding master here is an issue until these files are in master. Same for PS5 and PS4 file